### PR TITLE
sys-devel/llvm: fix git-format-clang shebang

### PR DESCRIPTION
**Pull request by austin987**
Fixes https://bugs.gentoo.org/show_bug.cgi?id=562688

### CODE REVIEW

**Comment by austin987** [comment]
@williamh @mgorny

File: sys-devel/llvm/llvm-9999.ebuild [hunk 1]
```diff
@@ -189,8 +189,13 @@ src_prepare() {
...
 		# Fix WX sections, bug #421527
 		find "${S}"/projects/compiler-rt/lib/builtins -type f -name \*.S -exec sed \
-			 -e '$a\\n#if defined(__linux__) && defined(__ELF__)\n.section .note.GNU-stack,"",%progbits\n#endif' \
-			 -i {} \; || die
+			-e '$a\\n#if defined(__linux__) && defined(__ELF__)\n.section .note.GNU-stack,"",%progbits\n#endif' \
+			-i {} \; || die
+
+		# Fix git-clang-format shebang, bug #562688
+		sed -e 's%#!/usr/bin/env python%#!/usr/bin/env python2.7%' \
+			-i  tools/clang/tools/clang-format/git-clang-format || die
```
Comment (mgorny) [inline sys-devel/llvm/llvm-9999.ebuild]: There's `python_fix_shebang` function.

**Comment by mgorny** [comment]
Also, please revbump only the newest ~arch and stable (3.7.1 and 3.8.0). Other versions you don't have to fix even.

**Comment by austin987** [comment]
@mgorny okay, updated

**Comment by mgorny** [comment]
Thanks. I've cherry-picked it locally and will push soon along with a few other fixes.

**Pull request closed without merging**